spacy_transformers/layers/hf_shim.py

```diff
@@ -61,6 +62,12 @@ def to_bytes(self):
             config = hf_model.transformer.config.to_dict()
             tokenizer = hf_model.tokenizer
             with make_tempdir() as temp_dir:
+                if hasattr(tokenizer, "vocab_file"):
+                    vocab_file_name = tokenizer.vocab_files_names["vocab_file"]
+                    vocab_file_path = str((temp_dir / vocab_file_name).absolute())
+                    with open(vocab_file_path, "wb") as fileh:
+                        fileh.write(hf_model.vocab_file_contents)
+                    tokenizer.vocab_file = str((temp_dir / vocab_file_name).absolute())
```

Comment:
Ah never mind I think I got it :-)

Reply:
Yes, you have to configure the tokenizer so that `tokenizer.save_vocabulary` knows where to look for the file. Since it's already in the final location, I think it even skips the copying.